msos/MixedStack.cs

```diff
@@ -312,5 +404,256 @@ public void PrintStackTrace(CommandExecutionContext context, uint index)
+
+        private static UnifiedBlockingReason ConvertToUnified(string objectType)
+        {
+            UnifiedBlockingReason result = UnifiedBlockingReason.Unknown;
+            switch (objectType)
+            {
+                case "Thread":
+                    result = UnifiedBlockingReason.Thread;
+                    break;
+                case "Job":
+                    result = UnifiedBlockingReason.Job;
+                    break;
+                case "File":
+                    result = UnifiedBlockingReason.File;
+                    break;
+                case "Semaphore":
+                    result = UnifiedBlockingReason.Semaphore;
+                    break;
+                case "Mutex":
+                    result = UnifiedBlockingReason.Mutex;
+                    break;
+                case "Section":
+                    result = UnifiedBlockingReason.CriticalSection;
+                    break;
+                case "Mutant":
+                    result = UnifiedBlockingReason.Mutex;
+                    break;
+                case "ALPC Port":
+                    result = UnifiedBlockingReason.Alpc;
+                    break;
+                case "Process":
+                    result = UnifiedBlockingReason.ProcessWait;
+                    break;
+                case "Unknown":
+                    result = UnifiedBlockingReason.Unknown;
+                    break;
+                case "None":
+                    result = UnifiedBlockingReason.None;
+                    break;
+                case "Timer":
+                    result = UnifiedBlockingReason.Timer;
+                    break;
+                case "Event":
+                    result = UnifiedBlockingReason.Event;
+                    break;
+                    //case "Callback": break;
+                    //case "Desktop": break;
+                    //case "Key": break;
+                    //case "IoCompletion": break;
+                    //case "Directory": break;
+                    //case "WindowStation": break;
+                    //case "WaitCompletionPacket": break;
+                    //case "TpWorkerFactory": break;
+                    //case "Timer": break;
+            }
+            return result;
+        }
+
+    }
+
+
+    public enum UnifiedBlockingReason
+    {
+        //Based on ClrThread BlockingReason Enumerations
+        None = 0,
+        Unknown = 1,
+        Monitor = 2,
+        MonitorWait = 3,
+        WaitOne = 4,
+        WaitAll = 5,
+        WaitAny = 6,
+        ThreadJoin = 7,
+        ReaderAcquired = 8,
+        WriterAcquired = 9,
+
+        //Based on WCT_OBJECT_TYPE Enumerations
+        CriticalSection = 20,
+        SendMessage = 11,
+        Mutex = 12,
+        Alpc = 13,
+        Com = 14,
+        ThreadWait = 15,
+        ProcessWait = 16,
+        Thread = 17,
+        ComActivation = 18,
+        UnknownType = Unknown,
+        File = 19,
+        Job = 20,
```

Comment:
Having the value 20 for both `CriticalSection` and `Job` doesn't sound like a good idea.